.github/workflows/check-codebase.yml

```diff
@@ -0,0 +1,95 @@
+name: Check codebase
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
```

Comment:
Using `--frozen-lockfile`  instead of `--pure-lockfile` according to https://classic.yarnpkg.com/en/docs/cli/install/

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass `--frozen-lockfile` flag.